auth.go

```diff
@@ -14,7 +14,14 @@ import (
 	"strings"
 	"time"
 
+	jcrypto "github.com/SermoDigital/jose/crypto"
```

Comment:
can you please explain why you chose this library? Did you have any other alternative?

Reply:
I chose it from https://jwt.io/#libraries, where there are lot of JWT library alternatives. Among all of Golang's, this library is the one with most star while still has all features listed included. There is one with more star but does not support issuer check and some other function.